### PR TITLE
Add contact_map

### PR DESCRIPTION
**Pull request by dwhswenson**
Adding my new `contact_map` repo to conda-forge, now that it is on PyPI. My first conda-forge contribution! Let's see if I got it to work....

### CODE REVIEW

File: recipes/contact_map/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - contact_map
+
+about:
+  home: http://github.com/dwhswenson/contact_map
+  license: LGPL-2.1+
+  license_family: LGPL
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: LICENSE.txt
```
Comment (synapticarbors) [inline recipes/contact_map/meta.yaml]: It doesn't look like the license is in the upstream `MANIFEST.in`, so it's not being packaged with the source distribution. You can manually package it by adding a copy of the license to this directory and then:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

File: recipes/contact_map/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/contact_map/meta.yaml]: When you use `--single-version-externally-managed`, you only typically need `python` and `setuptools` as `build` requirements (unless `setup.py` imports other modules).

File: recipes/contact_map/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+{% set name = "contact_map" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "695967007dc43b639efd458130411463ce79893863699d2d32b60c1b848cd062" %}
+# sha256 comes from
```
Comment (synapticarbors) [inline recipes/contact_map/meta.yaml]: Please delete comments.

File: recipes/contact_map/meta.yaml (hunk 1 above)
Comment (dwhswenson) [reply recipes/contact_map/meta.yaml]: Thanks -- the better approach would be to update the `MANIFEST.in`, right? 

This 0.1.x series is mainly for me to play with getting distribution/deployment set up, so I'm just as happy to bump to 0.1.2 on PyPI (including adding `LICENSE` to the sdist) and have that be the first conda-forge release.

Other changes have been implemented already; I'll push those when I have a release that includes the license file on PyPI.

**Comment by dwhswenson** [comment]
@synapticarbors : the build for this on master seems to have failed (https://travis-ci.org/conda-forge/staged-recipes/builds/285250016) leaving an empty feedstock repository (https://github.com/conda-forge/contact_map-feedstock). It looks like the problem may be that it was "Unable to register the repo on Travis".

I assume the recipe doesn't cause it. The underscore in the name isn't a problem, is it? (I see other feedstocks with that naming convention.)

Does this just need a re-run of the Travis run? (which needs to be done by someone with appropriate permissions) Or will it automatically get re-run the next time a recipe is merged?

Mainly, is there anything that ***I*** need to do?

**Comment by synapticarbors** [comment]
@dwhswenson - I'm not sure what the problem is and won't be able to look into it promptly, so I'm pinging @conda-forge/staged-recipes to see if anyone has a solution.

**Comment by dwhswenson** [comment]
Looks like it got re-run, and that fixed it!

**Comment by jakirkham** [comment]
Sorry for the delay! Sometimes we have issues with Travis CI's API (given our org's size 😜). I'm guessing that is what you saw. Normally a restart of Travis CI is sufficient to clear things out.

Though there isn't always someone who has time and knowledge to diagnose this is the case and restart Travis CI. So we are planning on automating that a bit with PR ( https://github.com/conda-forge/staged-recipes/pull/4096 ). This should help speed up the clearance rate.

**Pull request merged**